deploy/cluster-operator/templates/service_account.yaml

```
@@ -1,8 +1,51 @@
-{{- $fullName := include "cluster-operator.fullname" . -}}
 apiVersion: v1
 kind: ServiceAccount
 metadata:
-  name: {{ $fullName }}
+  name: cluster-operator
+  namespace: {{ .Release.Namespace }}
   labels:
-{{ include "cluster-operator.labels" . | indent 4 }}
+imagePullSecrets:
+  - name: infobloxctokey
+
 ---
+apiVersion: rbac.authorization.k8s.io/v1beta1
+kind: ClusterRole
+metadata:
+  name: cluster-operator
```

```suggestion
  name: {{ include "cluster-operator.fullname" . }}
```

All of the objects need to include the default labels template found in the chart https://github.com/infobloxopen/cluster-operator/pull/43/files?file-filters%5B%5D=.go&file-filters%5B%5D=.yaml&file-filters%5B%5D=No+extension#diff-1af5202ce72159d53ed4095eed8d2deaL6-L7